src/dashboards/videos/course.jsonnet

```diff
@@ -0,0 +1,317 @@
+      legend: {
+        calcs: ['count', 'max', 'mean'],
+        displayMode: 'list',
+        placement: 'bottom',
+      },
+    },
+    pluginVersion: '8.0.3',
+    targets: [
+      {
+        alias: '',
+        bucketAggs: [
+          {
+            field: 'timestamp',
+            id: '2',
+            settings: {
+              interval: '3600000',
+              min_doc_count: '1',
+            },
+            type: 'histogram',
+          },
+          {
+            field: common.field.actor_account_name,
+            id: '3',
+            settings: {
+              min_doc_count: '1',
+              order: 'desc',
+              orderBy: '_term',
+              size: '0',
+            },
+            type: 'terms',
+          },
+        ],
+        metrics: [common.object.count_metric],
+        query: common.query.school_course_session,
+        refId: 'A',
+        timeField: 'timestamp',
+      },
+    ],
+    transformations: [
+      {
+        id: 'groupBy',
+        options: {
+          fields: {
+            Count: {
+              aggregations: [
+                'sum',
+              ],
+              operation: 'aggregate',
+            },
+            [common.field.actor_account_name]: {
+              aggregations: [],
+              operation: 'groupby',
+            },
+          },
+        },
+      },
+    ],
+    type: 'histogram',
+  },
+  gridPos={ x: 12, y: 18, w: 12, h: 9 }
+)
+.addPanel(
+  {
+    title: 'Completed videos by user',
+    description: |||
+      The distribution of the number of times users completed videos.
+      On the X-axis is the number of completed videos.
+      On the Y-axis is the number of users.
+    |||,
+    datasource: common.lrs,
+    fieldConfig: {
+      defaults: {
+        color: {
+          mode: 'palette-classic',
+        },
+        displayName: 'User',
+        mappings: [],
+      },
+      overrides: [],
+    },
+    options: {
+      bucketOffset: 0,
+      legend: {
+        calcs: ['count', 'max', 'mean'],
+        displayMode: 'list',
+        placement: 'bottom',
+      },
+    },
+    pluginVersion: '8.0.3',
+    targets: [
+      {
+        alias: '',
+        bucketAggs: [
+          {
+            field: 'timestamp',
+            id: '2',
+            settings: {
+              interval: '3600000',
+              min_doc_count: '1',
+            },
+            type: 'histogram',
+          },
+          {
+            field: common.field.actor_account_name,
+            id: '3',
+            settings: {
+              min_doc_count: '1',
+              order: 'desc',
+              orderBy: '_term',
+              size: '0',
+            },
+            type: 'terms',
+          },
+        ],
+        metrics: [common.object.count_metric],
+        query: '%(course_query)s AND verb.id:"%(completed)s"' % {
+          course_query: common.query.school_course_session,
+          completed: common.value.verb_id_completed,
+        },
+        refId: 'A',
```

Comment:
We can do better than this default :sweat_smile:

Reply:
I'm not sure, I think we have to choose a value between `A-Z` ? Found [this old grafana post](https://community.grafana.com/t/dashboard-import-refid/3651).

Reply:
I thought it should be a unique identifier that could be personalized.

Reply:
Just take into account that queries named with "NA" won't be supported, plus the max of queries is about 26 (https://github.com/grafana/grafana/issues/28104#issuecomment-705488686) 
but if we personnalized them, we won't named them "NA" :stuck_out_tongue: